README.md

```diff
@@ -121,19 +96,90 @@ the Remote Monitoring WebApp
 }
 ```
 
-### Verify the Web UI and Microservices are deployed
+#### Verify the Web UI and Microservices are deployed
 
 1. Click on the link that is shown in the output window, it will take you to
    the Remote Monitoring WebApp
 1. It can take upto 5 minutes for the webapp to be ready
 1. Go to {azurewebitesurl}/hubmanager/v1/status to see HubManager microservice status
 1. Go to {azurewebitesurl}/devices/v1/status to see Devices microservice status
 
-## Local
-Please look [here](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Running-the-Remote-Monitoring-Solution-Locally) for more information for using this option
+### Local Deployment
+Please look [here](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-deploy-local for more information for using this option)
 1. `pcs -s local`
 1. Follow onscreen prompts to start the deployment
 
+Deployment Options
+==================
+
+## Overview
+
+When you deploy the preconfigured solution, there are several options that configure the deployment process:
+
+| Option | Values | Description |
+| ------ | ------ | ----------- |
+| SKU    | `basic`, `standard`, `local` | A _basic_ deployment is intended for test and demonstrations, it deploys all the microservices to a single virtual machine. A _standard_ deployment is intended for production, it deploys the microservices to multiple virtual machines. A _local_ deployment configures a Docker container to run the microservices on your local machine, and uses Azure services, such as storage and Cosmos DB, in the cloud. |
+| Runtime | `dotnet`, `java` | Selects the language implementation of the microservices. |
+
+To learn about how to use the local deployment, see [Running the remote monitoring solution locally](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Running-the-Remote-Monitoring-Solution-Locally#deploy-azure-services-and-set-environment-variables).
+
+## Basic
+
+The purpose of the basic deployment is to demo the capabilities of the system
+and requires minimal setup, deploying all resources to a single VM.
+
+Creating a Basic solution will result in the following Azure services being

```

Comment:
s/Creating/Deploying/

Reply:
nit: need consistency, either "basic" and "standard" or "Basic" and "Standard" across the entire doc